ts/lib/keys.ts

```diff
@@ -121,13 +121,36 @@ export function isArrowDown(event: KeyboardEvent): boolean {
     return isApplePlatform() && metaPressed(event) && event.code === "KeyN";
 }
 
-export function onEnterOrSpace(callback: () => void): (event: KeyboardEvent) => void {
-    return (event: KeyboardEvent) => {
-        switch (event.code) {
-            case "Enter":
-            case "Space":
-                callback();
-                break;
+/**
+ * Builds function for `on:keydown` event to call `callback` if a key from `keys` was pressed
+ */
+export function onSomeKey(callback: () => void, ...keys: string[]): (event: KeyboardEvent) => void {
+    // I really want to generalize this as some structure of ([list of key combinations], function) pairs
```

Comment:
I recommend reverting this change. We're not ready to introduce custom keyboard shortcuts yet and only have one caller, so this doesn't meet the [rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)), and this function can't handle things like modifier keys.